src/OpenTelemetry.SemanticConventions/scripts/templates/SemanticConventionsAttributes.cs.j2

```diff
@@ -0,0 +1,91 @@
+{%- macro print_value(type, value) -%}
+  {{ "\"" if type == "string"}}{{value}}{{ "\"" if type == "string"}}
+{%- endmacro %}
+
+{%- macro format_remarks(text) -%}
+{%- set notes = '\n        /// '.join(text.splitlines()).encode('ascii', 'xmlcharrefreplace').decode() -%}
+{{notes}}
+{%- endmacro -%}
+
+{%- macro format_xml_doc(text) -%}
+{%- set escaped = text.encode('ascii', 'xmlcharrefreplace').decode() -%}
+  {%- if not escaped.endswith('.')-%}
+    {{escaped + '.'}}
+  {%- else -%}
+    {{escaped}}
+  {%- endif -%}
+{%- endmacro -%}
+
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+// <auto-generated>This file has been auto generated from scripts{{template}}</auto-generated>
+
+#pragma warning disable CS1570 // XML comment has badly formed XML
+
+namespace OpenTelemetry.SemanticConventions
+{
+    /// <summary>
+    /// Constants for semantic attribute names outlined by the OpenTelemetry specifications.
+    /// </summary>
+    /// <remarks>
+    /// Schema and specification version: {{schemaUrl}}.
```

Comment:
Once .NET allows to set schema URL, having a single one would be problematic - different instrumentations can produce different schema versions.